multiworld/envs/mujoco/sawyer_reach_torque/sawyer_reach_torque_env.py

```diff
@@ -332,14 +266,21 @@ def compute_rewards(self, achieved_goals, desired_goals, info):
             r = -(distances < self.indicator_threshold).astype(float)
         else:
             raise NotImplementedError("Invalid/no reward type.")
-        assert (r<=0).all()
         return r
 
-    # def get_env_state(self):
-    #     base_state = self._get_env_obs()
-    #     goal = self._state_goal.copy()
-    #     return base_state, goal
+    def set_to_goal(self, goal):
+        pass
```

Comment:
You should do something like `raise NotImplemented()` to avoid having these silent bugs where someone assumes that this function works.